polybius.py

```diff
@@ -105,12 +103,39 @@ def one_pad_crypto_technique(coordinates, one_time_key):
 
     return(final_ciphertext)
 
+def reverse_one_time_pad(ciphertext, key):
+    # xors every 2 digit number in cipher text with key
+    ret = ''
+    key = int(key)
```

Comment:
shouldn't this be binary? or does the ^ Xor operator automatically convert to binary?